Userland/Demos/Mouse/main.cpp

```diff
@@ -175,7 +175,7 @@ class MainFrame final : public GUI::Frame {
     unsigned m_buttons { 0 };
     unsigned m_wheel_delta_acc { 0 };
     bool m_show_scroll_wheel { false };
-    unsigned m_double_click_buttons { 0 };
+    unsigned m_double_clicked_buttons { 0 };
```

Comment:
This is actually the same thing as for the initialized members fix, as you did it one way in a previous commit, and then you correct it in a latter one. You should be able to squash this commit into the one were you introduced this variable.

Also, I feel like m_double_clicked_button (singular) would be best.

Thanks for the fixes!

Reply:
Sorry about that, I pushed the change squashes the fixup commit. Thanks for your patience :) 

I don't agree that singular is better, as multiple simultaneous double clicks are possible (and supported), and the similar `m_buttons` is plural.

Reply:
Cool! Up to you I guess :)